src/read.c

```diff
@@ -3389,6 +3389,41 @@ static avifResult avifDecoderFlush(avifDecoder * decoder)
     return AVIF_RESULT_OK;
 }
 
+// Searches for the primary color item or the auxiliary alpha item.
+// Returns the target item if found, or NULL.
+static avifDecoderItem * avifDecoderFindItem(avifDecoder * decoder, avifBool alpha, uint32_t parentItemID)
```

Comment:
Optional: This function can take `avifDecoderData * data` as the first parameter and be renamed `avifDecoderDataFindItem`.